<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Setting , #listWidget
{
	background-color: rgb(247,250,253);
}
QStackedWidget#stackedWidget ,#pageSetThreadNum, #pageSetFileFilters, #frame_pBug
{
	background-color: rgb(255,255,255);
}
Line
{
	background-color: rgb(235,235,236);
}
QLabel#labelCheckType , #label_Warring
{
	color: rgb(255, 0, 0);
}

QSlider {
    padding: 2px;
    height: 40px;
}
QSlider::groove:horizontal 
{
    subcontrol-origin: content;
    background: rgb(76, 76, 76);
    height: 20px;
	border-radius: 5px;
}
QSlider::handle:horizontal 
{
    background-color: rgb(50, 50, 50);
    width: 30px;
    border-radius: 15px;
    margin: -5px 0;
}
QSlider::sub-page:horizontal
{
    background: #999;
    margin: 5px;
    border-radius: 5px;
}
QSlider::add-page:horizontal
{
    background: #666;
    margin: 5px;
    border-radius: 5px;
}

QScrollBar::vertical {
    width: 8px;
	border-color: rgb(217,217,217);
}

QScrollBar::handle:vertical {
    min-height: 8px;
    border-width: 4px 0px 4px 0px;
    border-image:url(:/img/image/ScrollBar/vertical-handle.png) 17 0 17 0 repeat repeat;
}

QScrollBar::sub-line:vertical {
    width: 0px;
    height: 0px;
}
QScrollBar::add-line:vertical {
    width: 0px;
    height: 0px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../res/resource.qrc">:/img/image/Setting_new.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>SetThreadNum</string>
            </property>
            <property name="icon">
             <iconset resource="../res/resource.qrc">
              <normaloff>:/img/image/settinghold.png</normaloff>:/img/image/settinghold.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SetFileFilters</string>
            </property>
            <property name="icon">
             <iconset resource="../res/resource.qrc">
              <normaloff>:/img/image/FileFilters.png</normaloff>:/img/image/FileFilters.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QWidget" name="pageSetThreadNum">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>131</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Set Check Thread Num</string>
           </property>
          </widget>
          <widget class="QSlider" name="horSlidThreadNum">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>441</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_ThreadNum">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Thunder Num: %1</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pageSetFileFilters">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1505</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="lineEdit">
                   <property name="placeholderText">
                    <string>Find or join a custom file format</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_Warring">
                 <property name="text">
                  <string>Please directly enter the file format, such as ReadMe.md , then enter md .</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidget_2">
                 <item>
                  <property name="text">
                   <string>All Files ( *.* )</string>
                  </property>
                  <property name="checkState">
                   <enum>Unchecked</enum>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>txt</string>
                  </property>
                  <property name="checkState">
                   <enum>Unchecked</enum>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>doc</string>
                  </property>
                  <property name="checkState">
                   <enum>Unchecked</enum>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>exe</string>
                  </property>
                  <property name="checkState">
                   <enum>Unchecked</enum>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_pBut">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9,0,1">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelCheckType">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QPushButton" name="pButOK">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pButCancel">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
